### ISSUE TITLE
Support Vue3

### ISSUE THREAD

**Issue by ionutgit** [seeratawan01/vue-lazytube#12]
Good library! I'd like to use it, but my project is on vue 3. It would be nice if you can make this library available for Vue 3. Thanks!

**Comment by seeratawan01** [comment]
@jmdmacapagal Yes, I will release vue 3 component soon, already working on it. Thanks

**Comment by jmdmacapagal** [comment]
@seeratawan01 thanks! looking forward to it

**Comment by hacknug** [comment]
@seeratawan01 let us know if there's anything we can do to help 👍

**Comment by NtchPlayer** [comment]
Hi !
Do you have any news about a support for vue3 ?

**Comment by shankhadevpadam** [comment]
@seeratawan01 any update on vue 3 support?
